apps/platform_runner/mix.exs

```diff
@@ -24,19 +24,21 @@ defmodule PlatformRunner.MixProject do
 
   defp deps do
     [
+      {:benchee, "~> 1.0", only: [:test, :integration]},
```

Comment:
The actual diff here is that we added `service_profile` and `service_aggregate` and sorted the list.